### PR TITLE
Add dashboard script with css

### PR DESCRIPTION
**Pull request by brkicadis**

### CODE REVIEW

File: css/simple.css [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+hr {
+    border: 0;
+    height: 1px;
+    background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0));
+    margin: 15px 0 15px 0;
+}
+
+.gateway {
+    font-family: 'Roboto', 'Helvetica', 'Arial', sans-serif;
```
Comment (pretzelhands) [inline css/simple.css]: You're using Roboto here but never importing the font. I would suggest either switching the font to something that's imported or importing Roboto separately.

File: css/simple.css [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+    font-weight: 600;
+    font-size: 30px;
+}
+
+.shop-version {
+    font-family: 'Roboto', 'Helvetica', 'Arial', sans-serif;
```
Comment (pretzelhands) [inline css/simple.css]: Same here. No Roboto to be found anywhere.

File: model/dashboard.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+def reformat_dictionary(dictionary):
+    new_dict = {}
+    for project, gateway in dictionary.items():
+        for gateway_dict in gateway:
+            for gateway_name, results in gateway_dict.items():
+                new_dict = add_to_dict_array(new_dict, gateway_name, {project: results})
+    return new_dict
+
+
+def create_report_file():
```
Comment (pretzelhands) [inline model/dashboard.py]: Can we consider implementing a template engine like [Jinja2](http://jinja.pocoo.org/docs/2.10/)? The pure HTML in strings is quite confusing to look at and leads to lots of branching that could be avoided.

If you need help in getting it set up, I'll be happy to assist!

File: css/simple.css (hunk 1 above)
Comment (brkicadis) [reply css/simple.css]: Imported from https://fonts.google.com/specimen/Roboto?selection.family=Roboto

File: css/simple.css (hunk 2 above)
Comment (brkicadis) [reply css/simple.css]: Imported from https://fonts.google.com/specimen/Roboto?selection.family=Roboto

**Pull request merged**